rpcs/ledgerService_test.go

```diff
@@ -228,6 +274,25 @@ type BasicRPCNode struct {
 	peers    []network.Peer
 }
 
+func (b *BasicRPCNode) Address() (string, bool) {
```

Comment:
is it possible to embed the MockNetwork here instead of re-implementing all of these ?